scripts/install-linux.sh

```diff
@@ -70,7 +72,9 @@ if ! command -v wget &> /dev/null; then
     if [ "$OS_TYPE" == "debian" ]; then
         echo "sudo apt install wget"
     elif [ "$OS_TYPE" == "redhat" ]; then
-            echo "sudo yum install wget"
+        echo "sudo yum install wget"
```

Comment:
```suggestion
        echo "sudo dnf install wget"
```

yum is just a alias for dnf now days